### PR TITLE
Dart code style (test description style)

### LINKED ISSUE #105: Test description style
As we see that tests descriptions style varies - we need to prepare a document with the test description style guide so our documentation is consistent.

### PR DESCRIPTION
**Pull request by illia-romanenko**
Fixes #105 

## Checklist

- [x] Documented

### CODE REVIEW

File: docs/10_dart_code_style.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    ```dart
+    test("should throw ArgumentError trying to create an instance with null CI client", 
+    ```
+3. Test description not inside Test Group starts with Capital letter:
+    ```dart
+    testWidgets("Can't create widget without data",
+   ```
+4. Group and test descriptions should not end with a dot (.).
+5. If method name (or named constructor) is used in test description it should start with dot (.)
+and should have parenthesis:
```
[the remark this answers is not in the document]
Comment (illia-romanenko) [reply docs/10_dart_code_style.md]: Thanks, pushed updated code.

File: docs/10_dart_code_style.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    ```
+6. Use Test Group descriptions with class under test name:
+    ```dart
+    group("ReceiveProjectMetricUpdates", () {
+       .....
+       test("loads all fields in the performance metrics", () {
+    ```
+7. Multiline test descriptions shouldn't be used as such tests can't 
+be started in [Intellij](https://youtrack.jetbrains.com/issue/WEB-44842).
+8. Prefer using double quotes.
```
Comment (vladlaba) [inline docs/10_dart_code_style.md:64]: Maybe we should add examples for these points as for others above? We can use the same style as the Effective Dart do. I.e., provide `bad` and `good` examples and use `DO`, `DO NOT`, etc. words.

Comment (illia-romanenko) [reply docs/10_dart_code_style.md]: Thanks @vladlaba - please check the new version and let me know what you think 💭

**Comment by illia-romanenko** [comment]
Thanks, guys!

**Pull request merged**